repose-aggregator/components/services/datastore-service/datastore-service-api/src/main/java/org/openrepose/core/services/datastore/DatastoreService.java

```diff
@@ -19,13 +19,18 @@
  */
 package org.openrepose.core.services.datastore;
 
+import org.openrepose.commons.utils.encoding.EncodingProvider;
+import org.openrepose.core.services.RequestProxyService;
 import org.openrepose.core.services.datastore.distributed.ClusterConfiguration;
 import org.openrepose.core.services.datastore.distributed.DistributedDatastore;
 
+import java.net.InetSocketAddress;
+
 /**
  * DatastoreService - service that manages the lifecycle and configuration of {@link Datastore}s
  */
-@SuppressWarnings("squid:RedundantThrowsDeclarationCheck") //We're defining the contract here, so the extra explicitness is warranted
+@SuppressWarnings("squid:RedundantThrowsDeclarationCheck")
+//We're defining the contract here, so the extra explicitness is warranted
```

Comment:
I don't know. The formatter just didn't want it on the same line. I'm good either way. Since you obviously have a preference, I'll change it out.